components/calendar/src/astronomy.rs

```diff
@@ -395,24 +394,31 @@ impl Astronomical {
         )
     }
 
-    /// Calculates the right ascension at a given [`Moment`] of UTC time of an object at latitude `beta` and longitude `lambda`; all angles are in degrees.
+    /// Calculates the right ascension at a given [`Moment`] of UTC time of an object at celestial latitude `beta` and celestial longitude `lambda`; all angles are in degrees.
```

Comment:
Let’s call this _ecliptic_ latitude and longitude. While Meeus primarily uses _celestial_ (or just unqualified) latitude and longitude, he does list the synonym _ecliptical_ in his definition (p. 409). This is now clearly the more current term (see the charts in https://ui.adsabs.harvard.edu/search/p_=0&q=%22ecliptic%20longitude%22&sort=date%20desc%2C%20bibcode%20desc vs. https://ui.adsabs.harvard.edu/search/q=%22celestial%20longitude%22&sort=date%20desc%2C%20bibcode%20desc&p_=0) and is going to induce far less confusion with the celestial pole and equator.

(Four places in this file.)